terraform/modules/happy-stack-helm-eks/ecr.tf

```diff
@@ -0,0 +1,10 @@
+module "ecr" {
+  source   = "git@github.com:chanzuckerberg/cztack//aws-ecr-repo?ref=v0.59.0"
+  for_each = local.service_definitions
+
+  name           = "${var.stack_name}/${local.tags.env}/${each.value.name}"
```

Comment:
Nit: it might make sense to reverse these since `env` is a larger scope (iee, env contains stacks and a stack cotains services... so `<env>/<stack>/<service>`

Reply:
This is a direct translation of what we have today, switching it over might create a consistency issue.

Reply:
Let's keep it consistent with the current setup, and we can make determination later.

Reply:
Sounds good 👍